diesel/src/macros/mod.rs

```diff
@@ -989,6 +989,17 @@ macro_rules! __diesel_table_query_source_impl {
 ///
 /// assert_eq!(implicit_on_clause_sql, explicit_on_clause_sql);
 /// # }
+///
```

Comment:
The example code block should end here. (Put ``` in this line)

Also as said in the gitter conversion. I think the example above would benefit from adding the actual table! calls. (I would add them in line 978 before the call to `joinable!`)

Reply:
Sorry for the late response, I missed that.

As of 4695673e434127b20deb2e4a05d179052c6e4321 it was fixed  but seems like git  didn't picked it up as changed. Sorry for that.

Here's how it renders on the latest commit (2f1575e4d6affea2844cf0b756e578127d202285):

![screenshot from 2018-02-04 00-28-48](https://user-images.githubusercontent.com/16531710/35772283-0463b1a2-0933-11e8-8d5b-9642b6e1374c.png)